lib/core/engine/iteration.js

```diff
@@ -179,6 +194,24 @@ class Iteration {
       }
       await this.engineDelegate.onStopIteration(browser, index, result);
 
+      /*
+       * Look through the scripts and try to do our post scripts.
+       */
+      if (this.scriptsByCategory) {
+        await browser.runScripts(
```

Comment:
This is _very_ horizontally narrow.  Can we please use our large monitors?  We don't have infinite vertical space!

Reply:
eslint (correctly!) says that we do not want lines longer than 80 characters, the standard terminal size. Because this line will end up being longer than 80 characters, it seems to aggressively add new lines between each of the parameters and line them up vertically. What do you suggest as the maximum line length? I can change that and commit it to the eslint rules in the repo so that this is not a problem in the future.